make-based/app-redis/files/.config

```diff
@@ -0,0 +1,375 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Unikraft/0.12.0~5bd4b94d Configuration
+#
+CONFIG_UK_FULLVERSION="0.12.0~5bd4b94d"
+CONFIG_UK_CODENAME="Epimetheus"
+CONFIG_UK_ARCH="x86_64"
+CONFIG_UK_BASE="/home/fotis/workspace/unikraft-root/unikraft"
+CONFIG_UK_APP="/home/fotis/workspace/unikraft-root/apps/app-redis"
```

Comment:
Let's remove these two: avoid hard-coded (and highly environment-specific) values which shouldn't be necessary anyway.
```suggestion
```